Examples/SampleApp/.gitignore

```diff
@@ -21,6 +21,11 @@ DerivedData
 *.ipa
 *.xcuserstate
 
+# Android
+#
+android/.gradle
+android/build
```

Comment:
`build` is already added in line 7
I think we should also use `.gradle` instead of `android/.gradle` similarly how we do it in main rn repo